public/api-spec.json

```diff
@@ -6236,223 +4859,167 @@
         }
       }
     },
-    "/reports/test-email": {
-      "post": {
-        "description": "Available to org admins only and with a valid license.\n\nYou need to have a permission with action `reports:send`.",
-        "tags": ["reports", "enterprise"],
-        "summary": "Send test report via email.",
-        "operationId": "sendTestEmail",
-        "parameters": [
-          {
-            "name": "body",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/CreateOrUpdateConfigCmd"
-            }
+    "/query-history": {
+      "get": {
+        "description": "Returns a list of queries in the query history that matches the search criteria.\nQuery history search supports pagination. Use the `limit` parameter to control the maximum number of queries returned; the default limit is 100.\nYou can also use the `page` query parameter to fetch queries from any page other than the first one.",
+        "tags": ["query_history"],
+        "summary": "Query history search.",
+        "operationId": "searchQueries",
```

Comment:
Query parameters are missing; [here](https://github.com/grafana/grafana/blob/a263a0f9bdc1a77b861032c160547aa4651de4e7/pkg/api/docs/definitions/search.go#L22) you can find an example definition

Reply:
Added in https://github.com/grafana/grafana/pull/50498/commits/c77dd362602cf796ceb40d0d8295cb37954e5de0